### PR TITLE
Fixing docs site home screen mobile layout issues

### PR DESCRIPTION
**Pull request by rusackas**
### SUMMARY

Home page wasn't working well on mobile. Tweaked the UI of the sideshow a bit to fit. It feels inadequate, but it works for the moment.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before:
image
image

After:
image
image

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: docs/src/pages/index.tsx [hunk 1]
```diff
@@ -405,7 +429,7 @@ const Theme = () => {
                   role="button"
                 >
                   <h2>Investigate</h2>
-                  <span>Use sqlab to write queries to explore your data</span>
+                  <span>Use SQL Lab to write queries to explore your data</span>
```
Comment (rusackas) [inline docs/src/pages/index.tsx:432]: @mistercrunch please confirm I've got the preferred capitalization form here.

**Comment by mistercrunch** [comment]
Ah, it was those big buttons carousel things!

**Comment by rusackas** [comment]
@pkdotson please have a look/review, this may collide a bit with #10854, but I hope not.

**Review by mistercrunch — APPROVED** [review]

**Comment by mistercrunch** [comment]
I re-triggered cypress on CI, flaky test is fixed in `master` if you want to rebase.

Oh, I think the `nowrap` I added on `<pre>` is not right btw, I think it makes blocks of code into a one-liner and that was not my intent...

**Pull request merged**